.github/workflows/tests.yml

```
@@ -0,0 +1,27 @@
+name: Python tests
+
+on:
+  push:
+    branches: 'master'
+  pull_request:
+    branches: 'master'
```

Restricting the branches is not always necessary. If you do this, you're losing the ability to run tests against commits pushed to topic branches w/o opening a PR for them.
```suggestion
  pull_request:
```